app/scripts/controllers/incoming-transactions.js

```diff
@@ -234,3 +265,20 @@ class IncomingTransactionsController {
 }
 
 module.exports = IncomingTransactionsController
+
+function pairwise (fn) {
```

Comment:
Something like `withPrevious` might be a better name? And then rename `cache` to `previousValue`. I think it'd be a bit more clear what it did then.